python/lsst/ts/linearstage/controllers/zaber_lst.py

```diff
@@ -312,48 +312,62 @@ async def disconnect(self) -> None:
                 await self.mock_server.close()
             self.mock_server = None
 
-    async def move_relative(self, value: float) -> None:
+    async def move_relative(self, value: float, axis: int) -> None:
```

Comment:
See the comment of `axis` above. But I saw the annotation is: `axis: Axis`. Maybe you need to verify and unify this.